include/vapor/VolumeParams.h

```diff
@@ -25,30 +25,70 @@ class PARAMS_API VolumeParams : public RenderParams {
 
     static string GetClassType() { return ("VolumeParams"); }
 
+    //! Get the current raytracing algorithm
+    //! \retval string - Current raytracing algorithm (Regular, Curvilinear, or Ospray) 
     std::string GetAlgorithm() const;
+
+    //! Set the current raytracing algorithm
+    //! \param[in] string - Raytracing algorithm (Regular, Curvilinear, or Ospray)
     void        SetAlgorithm(std::string algorithm);
+
     void        SetAlgorithmByUser(std::string algorithm);
     bool        GetAlgorithmWasManuallySetByUser() const;
     void        SetAlgorithmWasManuallySetByUser(bool v);
-
     static std::vector<float> GetSamplingRateMultiples();
+
+    //! Get the sampling rate multiplier used with the current raytracing algorithm
```

Comment:
Is this the sampling rate along the ray, or across the sampling plane? Please clarify. Also, should be "ray casting", not "ray tracing", correct?

Reply:
> Is this the sampling rate along the ray, or across the sampling plane?

I'm not sure.  @Stasj?

> Also, should be "ray casting", not "ray tracing", correct?

Yes, that is correct.  Fixed.

Reply:
The sampling rate is along the ray.